packages/password-input/README.md

```diff
@@ -0,0 +1,117 @@
+# Password Input
+
+![npm (scoped)](https://img.shields.io/npm/v/@leafygreen-ui/password-input.svg)
+
+#### [View on MongoDB.design](https://www.mongodb.design/component/password-input/example/)
+
+## Installation
+
+### Yarn
+
+```shell
+yarn add @leafygreen-ui/password-input
+```
+
+### NPM
+
+```shell
+npm install @leafygreen-ui/password-input
+```
+
+## Example
+
+```js
+import { PasswordInput } from '@leafygreen-ui/password-input';
+
+// Multiple notifications
+<PasswordInput
+  label="New password"
+  stateNotifications={[
```

Comment:
nit: did we intentionally lean away from verificationState

Reply:
this is the first i'm hearing about verificationState, it was stateValidations previously.